products/android.md

```diff
@@ -39,7 +39,7 @@ releases:
 
 -   releaseCycle: "10"
     codename: Queen Cake
-    eol: false
+    eol: true
```

Comment:
Can we put  a date here? Perhaps the date off the last security update?

Reply:
Well, I just followed the existing page, Android 9 and below are didn't mention EOL date.
If using the last Monthly Security Update date is fine, I can add them of course.

Reply:
Would be great IMHO.

I quickly tried to search changelogs / release notes / releases list for Android in the past but did not find anything. Can you leave a comment in the android.md front matter or here if you can find something @YandexStudio ?

Reply:
> Would be great IMHO.
> 
> I quickly tried to search changelogs / release notes / releases list for Android in the past but did not find anything. Can you leave a comment in the android.md front matter or here if you can find something @YandexStudio ?

As you may have heard, AOSP is a huge open source project with thousands of subprojects that anyone can contribute to, but the main reviewers are from Google. Strictly speaking, there is not an EOL date for each Android version, because committing to any old version at any time is allowed, but too old versions are often rarely have new patches submitted. The monthly Android security patch update is led by Google, which ensures the continuity of the update to some extent. Due to the huge AOSP project, since Google introduced the Android monthly security patch update mechanism in 2015, there is hardly any month when this phenomenon occurs - a supported Android version does not receive any security update. So we can conclude that when a new month's Android security bulletin is released and no longer provides any fixes for an old version of Android, it is when Google gives up on this Android version, and we can then say that this version of Android reaches EOL.

https://source.android.com/docs/security/bulletin